src/platform/site-wide/user-nav/components/SearchHelpSignIn.jsx

```diff
@@ -70,13 +69,17 @@ class SearchHelpSignIn extends React.Component {
     return (
       <div className="profile-nav">
         <SearchMenu
-          isOpen={this.props.isMenuOpen.search}
           clickHandler={this.handleSearchMenuClick}
+          isOpen={this.props.isMenuOpen.search}
         />
-        <HelpMenu
-          isOpen={this.props.isMenuOpen.help}
-          clickHandler={this.handleHelpMenuClick}
-        />
+        <a
+          className="vads-u-color--white vads-u-text-decoration--none vads-u-padding-top--0p5 vads-u-padding-x--1"
+          href="/contact-us"
```

Comment:
Can we add a trailing slash here -> `/contact-us/`? Also helps prevent a redirect